app/portainer/services/api/resourceControlService.js

```diff
@@ -30,15 +30,18 @@ angular.module('portainer.app')
 
   service.applyResourceControl = function(resourceControlType, resourceIdentifier, userId, accessControlData, subResources) {
     if (!accessControlData.AccessControlEnabled) {
-      return;
+      accessControlData.Ownership = 'public';
     }
 
     var authorizedUserIds = [];
     var authorizedTeamIds = [];
-    var administratorsOnly = false;
+    var publicOnly = false;
     switch (accessControlData.Ownership) {
       case 'administrators':
```

Comment:
I think that you should return in the `default` instead of adding an extra `adminitrators` case test here.